<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu_em</class>
 <widget class="QWidget" name="menu_em">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_return_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>430</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Arial&quot;;
font: 20pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>241</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_goods">
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>商品信息管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_order">
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>订单信息管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_user">
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>客户信息管理</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lb_welcome">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
